content/en/docs/concepts/scheduling-eviction/resource-bin-packing.md

```diff
@@ -3,58 +3,82 @@ reviewers:
 - bsalamat
 - k82cn
 - ahg-g
-title: Resource Bin Packing for Extended Resources
+title: Resource Bin Packing
 content_type: concept
 weight: 80
 ---
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
 
-The kube-scheduler can be configured to enable bin packing of resources along
-with extended resources using `RequestedToCapacityRatioResourceAllocation`
-priority function. Priority functions can be used to fine-tune the
-kube-scheduler as per custom needs. 
+In the scoring plugin `NodeResourcesFit` of kube-scheduler, there are two 
+scoring strategies that support the bin packing of resources: `MostAllocated` and `RequestedToCapacityRatio`.
 
 <!-- body -->
 
-## Enabling Bin Packing using RequestedToCapacityRatioResourceAllocation
+## Enabling Bin Packing using MostAllocated
+`MostAllocated` allows the users to specify the resources with weight for each resources to 
+score nodes on the request to capacity ratio. It favors nodes that have a high allocation of resources.
 
-Kubernetes allows the users to specify the resources along with weights for
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+profiles:
+- pluginConfig:
+  - args:
+      scoringStrategy:
+        resources:
+        - name: intel.com/foo
+          weight: 3
+        - name: intel.com/bar
+          weight: 3
+        type: MostAllocated
+    name: NodeResourcesFit
+```
+
+**This feature is disabled by default**
+
+## Enabling Bin Packing using RequestedToCapacityRatio
+
+`RequestedToCapacityRatio` allows the users to specify the resources along with weights for
 each resource to score nodes based on the request to capacity ratio. This
 allows users to bin pack extended resources by using appropriate parameters
-and improves the utilization of scarce resources in large clusters. The
-behavior of the `RequestedToCapacityRatioResourceAllocation` priority function
-can be controlled by a configuration option called `RequestedToCapacityRatioArgs`. 
-This argument consists of two parameters `shape` and `resources`. The `shape` 
+and improves the utilization of scarce resources in large clusters. It favors nodes according to a
+configured function of the allocated resources. The
+behavior of the `RequestedToCapacityRatio` in `NodeResourcesFit` score function
+can be controlled by a configuration option called `scoringStrategy`. 
+This argument consists of two parameters `requestedToCapacityRatioParam` and 
```

Comment:
```suggestion
Within the `scoringStrategy` field, you can configure two parameters: `requestedToCapacityRatioParam` and 
```

Reply:
And maybe you can link to https://kubernetes.io/docs/reference/config-api/kube-scheduler-config.v1beta3/#kubescheduler-config-k8s-io-v1beta3-ScoringStrategy